crates/rpc/rpc/Cargo.toml

```diff
@@ -91,4 +91,5 @@ optimism = [
     "reth-provider/optimism",
     "reth-rpc-eth-api/optimism",
     "reth-revm/optimism",
+    "reth-rpc-eth-types/js-tracer",
```

Comment:
this needs to be enabled on the reth-rpc-eth-types import in L33 directly

Reply:
Good point!